public/waitingroom.html

```
@@ -0,0 +1,164 @@
+        <div class="social-row">
+          <a href="#" class="s-link" title="Twitter/X">𝕏</a>
+          <a href="#" class="s-link" title="Discord">💬</a>
+          <a href="#" class="s-link" title="Instagram">📷</a>
+          <a href="#" class="s-link" title="Facebook">f</a>
+          <a href="#" class="s-link" title="YouTube">▶</a>
+          <a href="#" class="s-link" title="GitHub">⌘</a>
+        </div>
+        <p class="newsletter-label">SUBSCRIBE TO NEWSLETTER</p>
+        <div class="newsletter">
+          <input type="email" placeholder="Your email"/>
+          <button>➤</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)